lib/terraform/command_options.rb

```diff
@@ -56,6 +56,14 @@ def state=(value)
       store key: 'state', value: value
     end
 
+    def state_out
```

Comment:
This matches the tf option name but does it make more sense to have this named `state_out_path` to match its actual function?